pom.xml

```diff
@@ -1001,11 +1004,26 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <executions>
           <execution>
-            <id>run-integration-tests</id>
+            <id>run-minicluster-integration-tests</id>
             <goals>
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <skip>${accumulo.skipMiniClusterInFailsafe}</skip>
+              <groups>${accumulo.miniclusterTests}</groups>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
Ah, that may be the case, but multiple annotations is already problematic. I ran into an issue in master doing this with the perf tests. Suddenly, perf tests are being run when trying to run just the minicluster tests. My present approach is to only have one category per test presently (and override if necessary as some tests do).